awx/main/models/__init__.py

```diff
@@ -97,6 +98,93 @@
 User.add_to_class('accessible_objects', user_accessible_objects)
 
 
+def convert_jsonfields_to_jsonb():
```

Comment:
Where does this method get called from? The presentation of the large diff is messing me up.

And then after this gets called, it delays another method.